tests/utils_tests/test_functional.py

```diff
@@ -130,3 +131,11 @@ def test_lazy_equality(self):
 
         self.assertEqual(lazy_a(), lazy_b())
         self.assertNotEqual(lazy_b(), lazy_c())
+
+    def test_lazy_repr(self):
+        """
+        Tests that a proxy of lazy object's repr equals to the object's value
+        """
+        original_object = 'Lazy translation text'
+        lazy_obj = ugettext_lazy(original_object)
+        self.assertEquals(original_object, repr(lazy_obj))
```

Comment:
Yes definitely, thanks, changed it here and in the __repr__ method and works perfectly.